src/docker-osx-dev

```diff
@@ -379,13 +384,63 @@ function init_boot2docker {
     log_info "Starting Boot2Docker VM"
     boot2docker start --vbox-share=disable
   fi
+
+  configure_boot2docker_vars
+}
+
+################################################################################
+# docker-machine manipulation
+################################################################################
+
+function get_json_val () { 
+    python -c "import json,sys;sys.stdout.write(json.dumps(json.load(sys.stdin)$1))" | sed -e 's/^"//'  -e 's/"$//'
```

Comment:
Agreed! That is a hard one and I will have to think better on this. I saw it is possible to do with sed, grep and awk. 
Some examples that I saw using grep actually use GNU grep, which is not always available on OSX. I am open to any suggestion.